tests/test_transpilers_placer.py

```diff
@@ -21,36 +30,34 @@ def star_connectivity():
 
 def star_circuit():
     circuit = Circuit(5)
-    circuit.add(gates.CNOT(1, 0))
-    circuit.add(gates.CNOT(2, 0))
-    circuit.add(gates.CNOT(3, 0))
-    circuit.add(gates.CNOT(4, 0))
+    for i in range(1, 5):
+        circuit.add(gates.CNOT(i, 0))
     return circuit
 
 
-def test_check_placement_true():
+def test_assert_placement_true():
     layout = {"q0": 0, "q1": 1, "q2": 2, "q3": 3, "q4": 4}
     circuit = Circuit(5)
-    assert check_placement(circuit, layout, verbose=True)
+    assert assert_placement(circuit, layout, verbose=True)
```

Comment:
Since `assert_placement` is already raising errors, you do not have to `assert` before (you `assert` a `bool`, and you see from the chosen name that is now redundant ^^)